Dokan/Dokan/Scenes/ProductDetails/Views/InfoSellerView/InfoSellerView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  infoSellerView.swift
+//  Dokan
+//
+//  Created by Omar Ahmed on 21/07/2022.
+//
+
+import UIKit
+import UIDokan
+
+class InfoSellerView: UIView {
+    
+    @IBOutlet var contentView: UIView!
+    
+    @IBOutlet weak var sellerImageProfile: UIImageView!
+    
+    @IBOutlet weak var sellerName: UILabel!
+    
+    @IBOutlet weak var sellerType: UILabel!
+    
+    weak var delegate: InfoSellerViewDelegate?
+        
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+        addTapGestureRecognizer()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+        addTapGestureRecognizer()
+    }
+    
+    private func commonInit(){
+        Bundle.main.loadNibNamed("InfoSellerView", owner: self, options: nil)
+        addSubview(contentView)
+        contentView.frame = self.bounds
+        contentView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+        sellerType.addTrailing(image: UIImage(named: "verified"), text: sellerType.text! + " ")
```

Comment:
- Please stop force unwrap. 
- I think this should be moved to configure method?

```suggestion
        sellerType.addTrailing(image: UIImage(named: "verified"), text: sellerType.text ?? "" + " ")
```